stl/inc/chrono

```diff
@@ -2700,6 +2698,15 @@ namespace chrono {
         return {_STD move(_Leap_sec_info), _All_ls_positive};
     }
 
+    _NODISCARD inline string _Xtzdb_update_version(const string_view _Version, const size_t _Num_leap_seconds) {
+        const auto _Icu_end     = _Version.find_last_of('.');
+        const auto _Icu_version = _Icu_end == decltype(_Version)::npos
+                                    ? string{_Version}
+                                    : string{_Version.begin(),
+                                        _Version.begin() + static_cast<decltype(_Version)::difference_type>(_Icu_end)};

```

Comment:
Instead of `decltype(_Version)`, I think it would be simpler to say `string_view`. I don't think we need to worry about `_Version`'s type unexpectedly changing during refactoring.

Actually, we can use `substr`, see https://godbolt.org/z/hP4EoKEnc for an example of `str.substr(0, str.find_last_of('.'))`.